Files/Views/SettingsPages/About.xaml.cs

```diff
@@ -13,19 +12,16 @@ public About()
         {
             InitializeComponent();
             var version = Package.Current.Id.Version;
-            VersionNumber.Text = string.Format($"Version: {version.Major}.{version.Minor}.{version.Build}.{version.Revision}");
-        }
-
-        private async void OpenLogLocationButton_Click(object sender, RoutedEventArgs e)
-        {
-            await Launcher.LaunchFolderAsync(ApplicationData.Current.LocalFolder);
+            VersionNumber.Text = string.Format($"{ResourceController.GetTranslation("SettingsAboutVersionTitle")} {version.Major}.{version.Minor}.{version.Build}.{version.Revision}");
         }
+        
+        private void OpenLogLocationButton_Click(object sender, RoutedEventArgs e) => View_Models.SettingsViewModel.OpenLogLocation();
 
         private async void FeedbackListView_DoubleTapped(object sender, Windows.UI.Xaml.Input.DoubleTappedRoutedEventArgs e)
         {
             if (FeedbackListView.SelectedIndex == 0)
             {
-                await Launcher.LaunchUriAsync(new Uri(@"https://github.com/files-community/files-uwp/issues/new/choose"));
+                View_Models.SettingsViewModel.ReportIssueOnGitHub();
```

Comment:
Why did you move the code to SettingsViewModel?

Reply:
To keep it same across both the notification and about. In future if the link changes, we could change it easily from one place.